.gitlab-ci.yml

```diff
@@ -37,7 +37,7 @@ build:
         -e AWS_NETWORKING=true `
         -e SIGN_WINDOWS=true `
         -e NUGET_CERT_REVOCATION_MODE=offline `
-        registry.ddbuild.io/images/mirror/datadog/dd-trace-dotnet-docker-build:latest `
+        datadog/dd-trace-dotnet-docker-build:9-0-203 `
```

Comment:
Why did `registry.ddbuild.io/images/mirror/` change? Is it because the mirror one isn't updated yet?